index.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: page
+title: "FOSS Responders"
+ignore_title: true
+tags: ["foss responders", "floss responders", "oss", "floss", "foss", "open source", "aid", "coronavirus", "covid-19", "opensource"]
+excerpt: "Helping the Open Source ecosystem respond to Coronavirus"
+search_omit: true
+---
+
+<h1 class="center">FOSS Responders</h1>
+
+## Need help?
+
+If you work on open source are affected by Coronavirus, please get in touch. We are actively looking for ways to help out community members.
+
+<a href="https://github.com/foss-responders/support-requests/issues/new?assignees=&labels=&template=i-manage-events-or-communities-and-i-need-some-backup.md&title=%5BHELP%5D">
+  <span class="btn help">
+  I manage events or communities and I need some backup
+  </span>
+</a>
+
+<a href="https://github.com/foss-responders/support-requests/issues/new?assignees=&labels=&template=i-was-attending-an-event-that-was-cancelled-and-i-need-need-financial-aid.md&title=%5BINDIVIDUAL%5D">
+  <span class="btn help">
+  I was attending an event that was cancelled and I need financial aid
+  </span>
+</a>
+
+<a href="https://github.com/foss-responders/support-requests/issues/new?assignees=&labels=&template=we-had-to-cancel-our-event-and-we-need-financial-aid.md&title=%5BEVENT%5D">
+  <span class="btn help">
+  We had to cancel our event and we need financial aid
+  </span>
+</a>
+
+<a href="https://github.com/foss-responders/support-requests/issues/new?assignees=&labels=&template=we-need-people-to-help-us-organize-or-respond.md&title=%5BORGANIZE%5D">
+  <span class="btn help">
+  We need people to help us organize or respond
+  </span>
+</a>
+
+Have something else you need? Please [open an issue](https://github.com/foss-responders/support-requests/issues/new), [join our Slack](https://slack.opencollective.com/#crisis-working-group) or email [foss-responders@googlegroups.com](mailto:foss-responders@googlegroups.com). If you'd rather email than open a GitHub issue, please feel free to email, as well.
+
+## Conferences
+
+- Here is [a dynamic list of cancelled events](https://airtable.com/shrETNURgXNrGWbd8/tblc49hMMykARebo8). Not seeing an event? [Add it here](https://airtable.com/shr5QBJUPPOQUJfND).
+- If you were sponsoring a cancelled conference for recruitment purposes and you still want to connect with prospective candidates, Indeed is working on scheduling virtual hiring events. [Click here for more information](https://events.indeed.com/hiringevents/#get_started_form).
+
+What else should be here? [Get in touch](mailto:foss-responders@googlegroups.com!)
+
+
+## Funding
+
+- [Indeed](https://indeed.com) is running another round with $10k to focus on projects impacted by COVID-19. They're looking for others to join in organization, sponsorship, mentorship, and outreach.
```

Comment:
`[Indeed's Open Source Program Office](https://opensource.indeed.com) and the [Open Collective](https://opencollective.com) are organizing a FOSS Funders event in Q2 with a focus on projects impacted by COVID-19. They're looking for others to join in organization, sponsorship, mentorship, and outreach. Send an email to [opensource@indeed.com](mailto:opensource@indeed.com) for details.`